src/main/java/com/iexec/sms/secret/Secret.java

```diff
@@ -16,10 +16,7 @@
 
 package com.iexec.sms.secret;
 
-import lombok.AllArgsConstructor;
-import lombok.Data;
-import lombok.Getter;
-import lombok.NoArgsConstructor;
+import lombok.*;
```

Comment:
Shouldn't it be limited to only the removal of `lombok.Data` import here ?
Why do a global import and import unnecessary classes ?

Reply:
Sorry, it should be replaced with import `lombok.AccessLevel`.

Reply:
It's an IntellIj feature to keep imports short: when there are 4 or more imports from a same package, they are collapsed into a single `*` one. Looking at the web, there are pros and cons for that case.
As we don't currently have any strict coding rules, we may go for whatever you prefer.

Reply:
Fixed in https://github.com/iExecBlockchainComputing/iexec-sms/pull/134/commits/12e34cd7987a287063d3eb6bc730db7d0f69d0d1.